src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      {isMarried ? (
+        <p className="Person__partner">
+          {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
Prefer modifying tag content, not the whole tag:

```suggestion
      <p className="Person__partner">
      {isMarried ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}` : 'I am not married'
      </p>
```